backend/__fixtures__/leases.js

```diff
@@ -0,0 +1,104 @@
+//
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+'use strict'
+
+const createError = require('http-errors')
+const pathToRegexp = require('path-to-regexp')
+const { cloneDeep, find, merge } = require('lodash')
+const { POD_NAME, POD_NAMESPACE } = require('./env')
+
+const getLease = (renewTime = null) => {
+  if (!renewTime) {
+    renewTime = '1970-01-01T00:00:00.000000Z'
+  } else {
+    if (renewTime instanceof Date) {
+      renewTime = `${renewTime.toISOString().slice(0, -1)}000Z`
+    }
+  }
+
+  return {
+    apiVersion: 'coordination.k8s.io/v1',
+    kind: 'Lease',
+    metadata: {
+      creationTimestamp: '2023-01-01T01:01:01Z',
+      labels: {
+        app: 'gardener-dashboard',
+        role: 'dashboard'
+      },
+      namespace: POD_NAMESPACE,
+      name: 'gardener-dashboard-github-webhook',
+      uid: 'f810e8c6-e37f-4668-913d-89b37c598dd1'
+    },
```

Comment:
```suggestion
    metadata: {
      namespace: POD_NAMESPACE,
      name: 'gardener-dashboard-github-webhook'
    },
```
We only add the information we really need. Take a look at the other fixtures.